content/docs/started/workstation/getting-started-macos.md

```diff
@@ -0,0 +1,59 @@
++++
+title = "Kubeflow on MacOS"
+description = "Install Kubeflow on MacOS"
+weight = 2
++++
+
+<!--
+  TODO: Create a table that summarizes the options below, helping the user choose
+        more quickly
+  TODO: Surface the mac specific instructions here .. for instance, installing
+        vagrant and virtualbox through brew / brew cask
+-->
+
```

Comment:
Add an introductory sentence as a nice lead-in to this page, as you did in the Linux guide.

Reply:
Done.